kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/settings/SnapshotsTabUi.java

```diff
@@ -194,6 +194,11 @@ public void onSuccess(ArrayList<GwtSnapshot> result) {
         });
     }
 
+    @Override
+    public void clear() {
```

Comment:
Please change the header to: " * Copyright (c) 2011, 2020 Eurotech and/or its affiliates"